.docker/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+
+services:
+  app:
+    image: i24-reader:app-dev
+    build:
+      context: ../
+      dockerfile: .docker/app/Dockerfile
+      target: prod
+    command: 'yarn start'
```

Comment:
no need to set the command here if it does not differ from the one of the Dockerfile.